docs/reference/elasticsearch/security-privileges.md

```diff
@@ -1,17 +1,31 @@
 ---
+navigation_title: "Elasticsearch privileges"
 mapped_pages:
   - https://www.elastic.co/guide/en/elasticsearch/reference/current/security-privileges.html
+applies_to:
+  stack: all
+  serverless: all
 ---
 
-# Elasticsearch privileges [security-privileges]
+# {{es}} privileges [security-privileges]
 
-:::{note}
 This section provides detailed **reference information** for Elasticsearch privileges.
 
 Refer to [User roles](docs-content://deploy-manage/users-roles/cluster-or-deployment-auth/user-roles.md) in the **Deploy and manage** section for more information on how role-based access control works.
+
+
+:::{note}
+In {{serverless-full}}, Elastic manages the underlying infrastructure for you. Because of this, privileges related to cluster administration, scaling, snapshots, and availability are not available. Learn more about the [project custom roles managed by {{serverless-full}}](docs-content://deploy-manage/users-roles/serverless-custom-roles.md).
```

Comment:
putting the "how to" for serverless inside of this note makes serverless seem like a second class citizen. would pull out into the para(s) above, and use this link to instead explain about feature availability/features automated by serverless

Reply:
Sorry, only saw your comment after I merged. I can fix this tomorrow.